script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt

```diff
@@ -332,37 +356,43 @@ Before I can even finish my near-instantaneous movement―――
 [sha:c6d0ffa528ea5bda129cee4fa6283ecec494d48e]{
 -- Page 270, Offset 33964.
 -- 真下から突き上がる銃口は、惚れ惚れするぐらいの正確さで、俺の喉を串刺しにした。
-the muzzle, striking from directly below me, skewers my throat with admirable precision.
+// This line main driver for TLing 銃口 as spike
+The spike of her weapon slams up into me from below,
+ skewering my throat with admirable precision.
```

Comment:
Hmm, not sure gunpoint does it for me here. Like I can't come up with any sentence off the top of my head that isn't _at_ gunpoint, rather than using it as an actual noun.

Reply:
True enough. Though I think its intuitive regardless, and an interesting solution to a fun idiosyncrasy in the Japanese. Removing it entirely feels like it removes a bit of character. It weakens the notion that Seven is a gun, even if it shoots spike made out of bible pages.